src/loadDataset/index.ts

```diff
@@ -1,11 +1,14 @@
 import fs from 'fs'
+import pkgDir from 'pkg-dir'
 import { join, resolve } from 'path'
 import type prefixes from '../prefixes'
 
-function buildPath (prefix: keyof typeof prefixes) {
-  return resolve(join(__dirname, '../../ontologies', `${prefix}.nq`))
+async function buildPath (prefix: keyof typeof prefixes) {
+  const rootDir = await pkgDir(__dirname)
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  return resolve(join(rootDir!, 'ontologies', `${prefix}.nq`))
```

Comment:
The path is different in dev (`src`) and in deployed node_modules, when the JS file reside in package root.

Using `pkg-dir` seems like the easiest way to normalise the relative path to the package root dir